src/v/cluster/topics_frontend.cc

```diff
@@ -81,6 +82,13 @@ std::vector<std::string_view> get_enterprise_features(
     if (cfg.is_schema_id_validation_enabled()) {
         features.emplace_back("schema ID validation");
     }
+    if (const auto leaders_pref = cfg.cfg.properties.leaders_preference;
+        leaders_pref
```

Comment:
nit: I think our style is to explicitly check the result of `has_value()` on `std::optional`'s in conditions